lib/datadog/ci/test.rb

```diff
@@ -38,7 +38,10 @@ def self.set_tags!(span, tags = {})
         span.context.origin = Ext::Test::CONTEXT_ORIGIN if span.context
```

Comment:
⬆️  GitHub doesn't allow me to comment on line 34, but is it just me or is it kinda weird that this method can receive an arbitrary `tags` hash, but does not actually set all of the tags in it, instead just cherry picking a few?

Reply:
AFAIK it uses all `tags` passed in, but then adds/overwrites them with metadata. This should be an internal API anyway used only by the CI auto-instrumentation.